ribir/examples/storybook.rs

```diff
@@ -0,0 +1,166 @@
+      let on_primary: Brush = Palette::of(ctx).on_primary().into();
+    }
+    VScrollBar {
+
+      Column {
+        margin: EdgeInsets::horizontal(20.),
+        Column {
+          margin: EdgeInsets::only_top(20.),
+          Text::new("Button", &foreground, title_style.clone())
+          Row {
+            margin: EdgeInsets::only_top(20.),
+            FilledButton { svgs::ADD }
+            SizedBox { size: Size::new(20., 0.) }
+            FilledButton { Label::new("Filled button") }
+            SizedBox { size: Size::new(20., 0.) }
+            FilledButton {
+              color: Color::RED,
+              svgs::ADD
+              Label::new("Filled button")
+            }
+            SizedBox { size: Size::new(20., 0.) }
+            FilledButton {
+              color: Color::GREEN,
+              svgs::ADD
+              Label::new("Filled button")
+            }
+          }
+          Row {
+            margin: EdgeInsets::only_top(20.),
+            OutlinedButton { svgs::ADD }
+            SizedBox { size: Size::new(20., 0.) }
+            OutlinedButton { Label::new("Outlined button") }
+            SizedBox { size: Size::new(20., 0.) }
+            OutlinedButton {
+              color: Color::RED,
+              svgs::ADD
+              Label::new("Outlined button")
+            }
+            SizedBox { size: Size::new(20., 0.) }
+            OutlinedButton {
+              color: Color::GREEN,
+              svgs::ADD
+              Label::new("Outlined button")
+            }
+          }
+          Row {
+            margin: EdgeInsets::only_top(20.),
+            Button { svgs::ADD }
+            SizedBox { size: Size::new(20., 0.) }
+            Button { Label::new("Raw button") }
+            SizedBox { size: Size::new(20., 0.) }
+            Button {
+              color: Color::RED,
+              Label::new("Raw button")
+            }
+            SizedBox { size: Size::new(20., 0.) }
+            Button {
+              color: Color::GREEN,
+              Label::new("Raw button")
+            }
+          }
+          Row {
+            margin: EdgeInsets::only_top(20.),
+            FabButton { svgs::ADD }
+            SizedBox { size: Size::new(20., 0.) }
+            FabButton { Label::new("Fab button") }
+            SizedBox { size: Size::new(20., 0.) }
+            FabButton {
+              color: Color::RED,
+              svgs::ADD
+              Label::new("Fab button")
+            }
+            SizedBox { size: Size::new(20., 0.) }
+            FabButton {
+              color: Color::GREEN,
+              svgs::ADD
+              Label::new("Fab button")
+            }
+          }
+        }
+        Divider { extent: 30., end_indent: 150. }
+        Column {
+          Text::new("Lists", &foreground, title_style.clone())
+          Lists {
+            margin: EdgeInsets::only_top(20.),
+            ListItem {
+              Leading {
```

Comment:
Seems we can add convenient mini compose widget like `LeadingIcon`, `TrailingIcon`, `LeadingText`, `TraningText`.